assets/src/domain/eventEditor/ui/ticketAssignmentsManager/components/TicketAssignmentsManagerModal.tsx

```diff
@@ -15,18 +16,20 @@ const TicketAssignmentsManagerModal: React.FC = () => {
 	const submitButtonProps = useSubmitButtonProps();
 
 	return (
-		<Modal
-			bodyClassName='ee-ticket-assignments-manager__body'
-			cancelButtonProps={cancelButtonProps}
-			className='ee-ticket-assignments-manager'
-			isClosable={!submitButtonProps.isDisabled}
-			isOpen={true}
-			onClose={onCloseModal}
-			submitButtonProps={submitButtonProps}
-			title={title || __('Ticket Assignment Manager')}
-		>
-			<TicketAssignmentsManager />
-		</Modal>
+		<>
+			<Modal
+				bodyClassName='ee-ticket-assignments-manager__body'
+				cancelButtonProps={cancelButtonProps}
+				className='ee-ticket-assignments-manager'
+				isClosable={!submitButtonProps.isDisabled}
+				isOpen={true}
+				onClose={onCloseModal}
+				submitButtonProps={submitButtonProps}
+				title={title || __('Ticket Assignment Manager')}
+			>
+				<TicketAssignmentsManager />
+			</Modal>
+		</>
```

Comment:
don't need to wrap this in a fragment if there is only one element

Reply:
removed